site2/website-next/release-notes/all.md

```diff
@@ -16,7 +16,18 @@ slug: /
 [2.6.4](/release-notes/docs/pulsar-2.6.4)&ensp;&ensp;[2.6.3](/release-notes/docs/pulsar-2.6.3)&ensp;&ensp;[2.6.2](/release-notes/docs/pulsar-2.6.2)&ensp;&ensp;[2.6.1](/release-notes/docs/pulsar-2.6.1)&ensp;&ensp;  
 #### 2.5.x
 [2.5.2](/release-notes/docs/pulsar-2.5.2)&ensp;&ensp;[2.5.1](/release-notes/docs/pulsar-2.5.1)&ensp;&ensp;  
+
+#### Previous versions
+
+All release notes of previous versions are available at [here]().
```

Comment:
@urfreespace please add the link here when you add the "release notes of previous versions" page.